### PR TITLE
Add divider color functionality

### LINKED ISSUE #66: Improve consistency between WebAPK (PWA) and TWA Splash Screens
The generated splash screen is not very faithful to the original PWA splash screen.

Also, I see that the splash use the normal icon. I think it would be nice to use the maskable icon.

| PWA splash | Llama Pack splash |
|-------------|--------------------|
| Screenshot_20200115-114932 | Screenshot_20200115-115016 |

### PR DESCRIPTION
**Pull request by padr31**
Add support for changing navbar divider color in TWAs generated via bubblewrap

Users required the ability to change the color of the navigation bar
divider custom tabs, which in some cases looked inconsistent. After adding
this feature to chromium and androidX, this PR adds it into bubblewrap
via a navigationColor variable that can be set in the web-manifest.json

Test: Tested the integration of bubblewrap and android-browser-helper 
by building an example app and changing the navbar color. 
Also modified tests to account for new navbar color variables.

Closes #66

### CODE REVIEW

**Comment by andreban** [comment]
This seems to be solving #51 as well. Is that right?

**Comment by padr31** [comment]
> This seems to be solving #51 as well. Is that right?

Yes, I just noticed that android-browser helper is expecting to get a dark-mode color from bubblewrap, but it was not getting it, so I added that.

File: packages/core/src/lib/TwaManifest.ts [hunk 1]
```diff
@@ -120,6 +129,12 @@ export class TwaManifest {
     this.display = asDisplayMode(data.display!) || DEFAULT_DISPLAY_MODE;
     this.themeColor = new Color(data.themeColor);
     this.navigationColor = new Color(data.navigationColor);
+    this.navigationColorDark = new Color(data.navigationColorDark == undefined ?
+      DEFAULT_NAVIGATION_COLOR : data.navigationColorDark);
+    this.navigationDividerColor = new Color(data.navigationDividerColor == undefined ?
+      DEFAULT_NAVIGATION_DIVIDER_COLOR : data.navigationDividerColor);
+    this.navigationDividerColorDark = new Color(data.navigationDividerColorDark == undefined ?
+      DEFAULT_NAVIGATION_COLOR : data.navigationDividerColorDark);
```
Comment (PEConn) [inline packages/core/src/lib/TwaManifest.ts:137]: This can be `new Color(data.navigationDividerColorDark ?? DEFAULT_NAVIGATION_COLOR)`.

**Review by PEConn — APPROVED** [review]
Approved with one style comment.

**Comment by JudahGabriel** [comment]
@andreban how do we control dark mode splash screen color? Inside [TwaManifest](https://github.com/GoogleChromeLabs/bubblewrap/blob/master/packages/core/src/lib/TwaManifest.ts), the only dark variables I see are:

- [navigationColorDark](https://github.com/GoogleChromeLabs/bubblewrap/blob/master/packages/core/src/lib/TwaManifest.ts#L77)
- [navigationDividerColorDark](https://github.com/GoogleChromeLabs/bubblewrap/blob/master/packages/core/src/lib/TwaManifest.ts#L80)

Those affect the nav bar...do they also affect the splash screen?

**Comment by JudahGabriel** [comment]
Testing this on Android, those values only affect nav bar. Thus, unless I'm missing something, there isn't currently a way to change the splash screen color for dark mode. Let me know if I'm mistaken.

**Comment by andreban** [comment]
🤦  Yes, that's my mistake, that's dark mode for the navigation bar. Thanks for noticing and checking.

**Pull request merged**